slothy/helper.py

```diff
@@ -1338,6 +1338,7 @@ def run_code(code, txt=None):
                 log.error("Failed to emulate code using unicorn engine")
                 log.error("Code")
                 log.error(SourceLine.write_multiline(code))
+                raise SelfTestException("Selftest failed: Unicorn failed to emulate code")
```

Comment:
We should catch the exact exception and print the error information contained therein. Can be follow up.

Reply:
Pushed a commit addressing this  https://github.com/slothy-optimizer/slothy/pull/125/commits/4313b3de2d66b438bcd15ea4f901c8c036ca0d46